### PR TITLE
[#1] Allow for custom light and dark colours

### PR DESCRIPTION
**Pull request by VinSpee**
Create an options object which can take a light and dark colour:

``` js
postcss().use(yiq({
    dark: "#222222",
    light: "#efefef"
}))
```

let me know if you need any changes, as I haven't worked on a PostCSS plugin before.

Thanks!

### CODE REVIEW

**Comment by TrySound** [comment]
Tests should be passed. Also add new tests with different light and dark colors. And make both options unecessary like this

``` js
opts = opts || {};
opts.dark = opts.dark || '#000';
opts.light = opts.light || '#fff';
```

File: src/index.js [hunk 1]
```diff
@@ -9,13 +9,14 @@ function err (node) {
...
+function getYIQContrast (colour, colours) {
     let [r, g, b] = color(colour).rgbArray();
     let yiq = ((r * 299) + (g * 587) + (b * 114)) / 1000;
-    return (yiq >= 128) ? '#000' : '#fff';
+    return (yiq >= 128) ? colours.dark : colours.light;
 }
 
-export default postcss.plugin(plugin, () => {
+export default postcss.plugin(plugin, (opts) => {
+    if (typeof opts === 'undefined') opts = {dark: '#000000', light: '#ffffff'};
```
Comment (ben-eb) [inline src/index.js:19]: I would prefer using [object-assign](https://github.com/sindresorhus/object-assign) for this:

``` js
opts = assign({
    dark: '#000',
    light: '#fff'
}, opts);
```

**Comment by ben-eb** [comment]
Can you add some new tests for this behaviour, please? You can pass options in each test like so:

``` js
{
    message: 'should work with custom colours',
    fixture: 'h1{color:yiq;background:#000}',
    expected: 'h1{color:yellow;background:#000}',
    options: {light: 'yellow'}
}
```

**Comment by ben-eb** [comment]
We will also need API documentation for this feature. e.g:

https://github.com/ben-eb/postcss-reduce-idents#api

**Pull request merged**